api/db.py

```diff
@@ -49,21 +58,101 @@ def get_single_data(table, field, value):
     with get_db_cursor() as cur:
         cur.execute(sql)
         data = cur.fetchone()
+        if data is None:
+            return TypeError('No such data exists!')
     return data
 
 
-# get all known info about current minister or kns member
-def get_fully_today_member(query, value):
+# get all known info about current minister or knesset member
+def get_fully_today_member(query, value=None):
     try:
         with get_db_cursor() as cur:
             cur.execute(query, value)
             data = cur.fetchone()
-    except Exception:
-        return ValueError('No such member exist!')
+            if data is None:
+                raise ValueError('No such member exist!')
+    except Exception as e:
+        return e
```

Comment:
It's possible to only get `500` other than ValueError here.
So can I to just `return `value instead of `raise`?:
`return ValueError('No such member exist! Please enter another ID')`
and in `except Exception as e:` to do?:
logger.critical("critical error",e)